en/tutorials-and-examples/cms/database.rst

```diff
@@ -149,7 +149,7 @@ might look something like the following::
         // More configuration below.
     ];
 
-Once you've saved your **config/app.php** file, you should see that 'CakePHP is
+Once you've saved your **config/app.php** file, you should see that CakePHP is
 able to connect to the database' section have a green chef hat.
```

Comment:
The quoted string is  _‘CakePHP is able to connect to the database’_ so quote mark is not mistake.